app/src/main/java/com/iic/shopingo/ui/request_flow/views/CreateRequestItemListView.java

```diff
@@ -108,4 +123,8 @@ public void onItemEdited(View view, String value) {
     }
   }
 
+  public interface OnRequestItemListChanged {
+    public void onItemListChanged(View view);
```

Comment:
I don't like conventions for no reason.
I don't mind if you leave it there, since it won't be causing any issues, but it's not really needed (Which is why it won't cause any issues. There's absolutely nothing anyone can do with it).